torch/csrc/jit/codegen/cuda/test/test_gpu1.cpp

```diff
@@ -765,7 +765,7 @@ struct DummyExpr : public Expr {
       Val* _outrhs,
       Val* _lhs,
       Val* _rhs)
-      : Expr(passkey, ExprType::UnaryOp) // Not terribly safe...
+      : Expr(passkey) // terribly safe :-D
```

Comment:
I don't remember why it wasn't terribly safe, but I'm glad it is terribly safe now 😆